src/test/java/service/CheckoutControllerTest.java

```diff
@@ -39,7 +39,16 @@
 
     @BeforeClass
     public static void setupConfig() {
-        Application.gateway = BraintreeGatewayFactory.fromConfigFile(new File("config.properties"));
+        File configFile = new File("config.properties");
```

Comment:
It seems like we could use the code in the Factory instead of duplicating. Try catch on those methods would be nice.